### PR TITLE
Feat/support laravel 6

### PR DESCRIPTION
**Pull request by ReeceM**
This PR would allow a version of laravel-mongodb support installs on laravel 6.0.

It could be a different release tag or branch as it seems to have been done before.

Or possibly left until laravel 6.0 is more stable and the majority of user base has adopted and migrated to an updated codebase?

for #1821

(apologies for the first brief pr message..., I pressed enter to return the line.)

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,15 +11,15 @@
     ],
     "license" : "MIT",
     "require": {
-        "illuminate/support": "^5.8",
-        "illuminate/container": "^5.8",
-        "illuminate/database": "^5.8",
-        "illuminate/events": "^5.8",
+        "illuminate/support": "^6.0",
```
Comment (jenssegers) [inline composer.json]: Are there any super breaking changes that would make the package only support the latest version? Can we keep compatibility with 5.8?

File: composer.json [hunk 2]
```diff
@@ -11,15 +11,15 @@
...
+        "illuminate/container": "^6.0",
+        "illuminate/database": "^6.0",
+        "illuminate/events": "^6.0",
         "mongodb/mongodb": "^1.4"
     },
     "require-dev": {
-        "phpunit/phpunit": "^6.0|^7.0",
-        "orchestra/testbench": "^3.1",
+        "phpunit/phpunit": "^8.0",
```
Comment (jenssegers) [inline composer.json]: This will probably break travis tests on older PHP versions.

File: composer.json (hunk 1 above)
Comment (ReeceM) [reply composer.json]: Could this fix be put with a new 3.x release, as it seems if it requires the the ^5.8 branches it seems the composer install fails as it doesn't have the same versions?

Comment (jenssegers) [reply composer.json]: Why would it fail?

Comment (jenssegers) [reply composer.json]: Try `"^5.8|^6.0"`

File: composer.json (hunk 2 above)
Comment (ReeceM) [reply composer.json]: Okay, I see that it has done that already on this PR...

What would be the less breaking way of doing that?

Comment (jenssegers) [reply composer.json]: `"^6.0|^7.0|^8.0",` should work I think.

**Comment by Smolevich** [comment]
@jenssegers, @ReeceM, maybe delete changes  for tests in this PR and create in another PR?

**Comment by Smolevich** [comment]
https://github.com/jenssegers/laravel-mongodb/pull/1823 this PR with changes only composer.json

**Comment by jenssegers** [comment]
Supported added in 3.6.0

**Pull request closed without merging**